packages/framework/src/Twig/ImageExtension.php

```diff
@@ -156,7 +156,7 @@ public function getNoimageHtml(array $attributes = [])
     /**
      * @return string
      */
-    private function getEmptyImageUrl()
+    public function getEmptyImageUrl(): string
```

Comment:
please explain the change in the commit message or extract the change into standalone commit

Reply:
:+1: I will do the change on rebase to not mess with history right now

Reply:
in the end, no change that would deserve an additional description